website/source/docs/configuration/interpolation.html.md

```diff
@@ -312,6 +312,12 @@ The supported built-in functions are:
   * `pathexpand(string)` - Returns a filepath string with `~` expanded to the home directory. Note:
     This will create a plan diff between two different hosts, unless the filepaths are the same.
 
+  * `pow(x, y)` - Returns the base `x` of exponential `y`.
```

Comment:
@grubernaut from the above comment, it looks like support for float is not required here.